### PR TITLE
Edit profile page

### PR DESCRIPTION
**Pull request by amogiska**

### CODE REVIEW

File: crumbs/crumbs/PostsViewController.swift [hunk 1]
```diff
@@ -0,0 +1,29 @@
+//
```
Comment (tblake333) [inline crumbs/crumbs/PostsViewController.swift:1]: Delete

Comment (tblake333) [inline crumbs/crumbs/PostsViewController.swift:1]: Delete

File: crumbs/crumbs/ProfileViewController.swift [hunk 2]
```diff
@@ -14,15 +14,41 @@ class ProfileViewController: UIViewController {
...
     private final let POST_CARD_EMBED_SEGUE = "ProfileToCardSegue"
     private final let ABOUT_EMBED_SEGUE = "ProfileToAboutSegue"
-
+    
+    
+    @IBOutlet weak var edit: UIButton!
+    
+    
+    
+    @IBAction func EditButton(_ sender: Any) {
```
Comment (tblake333) [inline crumbs/crumbs/ProfileViewController.swift:23]: Delete

File: crumbs/crumbs/Base.lproj/Main.storyboard [hunk 3]
```diff
@@ -778,6 +865,23 @@
                                     <segue destination="a6W-Uu-ibI" kind="embed" identifier="ProfileToCardSegue" id="lff-NI-xgJ"/>
                                 </connections>
                             </containerView>
+                            <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" fixedFrame="YES" text="Label" textAlignment="center" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="fhV-Xy-EWa">
```
Comment (tblake333) [inline crumbs/crumbs/Base.lproj/Main.storyboard:868]: There seems to be a storyboard issue with this label. Is this a duplicate?

File: crumbs/crumbs/ProfileViewController.swift [hunk 4]
```diff
@@ -14,15 +14,41 @@ class ProfileViewController: UIViewController {
...
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        viewDidLoad()
+    }
+    
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+<<<<<<< HEAD
```
Comment (tblake333) [inline crumbs/crumbs/ProfileViewController.swift:35]: Remove this change

**Review by tblake333 — CHANGES_REQUESTED** [review]
Please fix the changes so the app can build

**Review by senkevinli — APPROVED** [review]

**Pull request merged**